docs/v1/P4Runtime-Spec.mdk

```diff
@@ -134,6 +165,35 @@ proto {
 }
 }
 
+@if html {
+prototext {
+  replace: "~ Begin Prototextblock&nl;\
+                 ````prototext&nl;&source;&nl;````&nl;\
+                 ~ End Prototextblock";
+  border: solid;
```

Comment:
I find a solid border to make code snippets it look like tables (obviously, this is very personal taste).
What other option is available?
How hard would it be include the generated html/pdf in the PR to visualize these changes?

Reply:
SG